File: Frontend/src/app/classes/acesso-aprovacoes/liberacao-acesso.ts
export class LiberacaoAcesso {
    codigo: number = 0;
    usuario: string = '';
    grupo: number = 0;
    tipoLiberacao: string = '';
    diaSemana: string = '';
    turno: number = 0;
    horaEntrada: Date = new Date();
    horaSaida: Date = new Date();
    bloquear: boolean = false;
    criadoEm: Date = new Date();
    atualizadoEm: Date = new Date();
}

export class Usuario { 
    id: string = '';
    nome: string = '';
}

File: Backend/TopSys.TopConWeb.Domain/Entities/LiberacaoAcesso/LiberacaoAcesso.cs
using System;

namespace TopSys.TopConWeb.Domain.Entities.LiberacaoAcesso
{
    public class LiberacaoAcesso
    {
        public int Codigo { get; set; }

        public string Usuario { get; set; }

        public int Grupo { get; set; }

        public string TipoLiberacao { get; set; }

        public string DiaSemana { get; set; }

        public int Turno { get; set; }

        public TimeSpan HoraEntrada { get; set; }

        public TimeSpan HoraSaida { get; set; }

        public Boolean Bloquear { get; set; }

        public DateTime? CriadoEm { get; set; }

        public DateTime? AtualizadoEm { get; set; }
    }
}
